src/backend/marsha/core/permissions.py

```diff
@@ -41,6 +41,10 @@ def has_permission(self, request, view):
         ):
             return True
 
+        # If the user is a staff user and the request is a GET, then always allow
+        if user.is_staff and request.method == "GET":
```

Comment:
Can you please create a dedicated permission class ? I don't like mixing several things in a permission class.